package.json

```diff
@@ -41,6 +41,7 @@
     "babel-core": "^5.8.21",
     "babel-eslint": "^3.1.17",
     "babel-loader": "^5.3.2",
+    "babel-runtime": "^5.8.25",
```

Comment:
I saw that all babel related dependencies are in the root package.json thats why decided to `babel-runtime` there as well. It works both ways, but depends what one finds more logical :)